### PR TITLE
ActiveForm::validateOnType: Avoids validation when shift, ctrl, alt and ...

### PR DESCRIPTION
**Pull request by sdlins**
...arrows are pressed [#6975]

### CODE REVIEW

**Comment by slinstj** [comment]
#6975

File: framework/assets/yii.activeForm.js [hunk 1]
```diff
@@ -416,7 +416,10 @@
             });
         }
         if (attribute.validateOnType) {
-            $input.on('keyup.yiiActiveForm', function () {
+            $input.on('keyup.yiiActiveForm', function (e) {
+                if ($.inArray(e.which, [16, 17, 18, 37, 38, 39, 40]) !== -1 ) {
```
Comment (samdark) [inline framework/assets/yii.activeForm.js]: Why comparing with -1?

**Comment by slinstj** [comment]
-1 = not found, 0 = first found

**Comment by samdark** [comment]
Merged. Thank you!

**Comment by samdark** [comment]
https://github.com/yiisoft/yii2/commit/41e5a8a83bd83f24e2abb14c7754596ba34685d0

**Pull request closed without merging**